res/css/views/dialogs/_DevtoolsDialog.scss

```diff
@@ -18,7 +18,11 @@ limitations under the License.
     margin: 10px 0;
 }
 
-.mx_DevTools_RoomStateExplorer_button, .mx_DevTools_RoomStateExplorer_query {
+.mx_DevTools_ServersInRoomList_button {
+    cursor: default !important;
```

Comment:
A comment on why we need `!important` would be great, as we should be avoiding it where possible.